lib/FormatMistake.vala

```diff
@@ -21,7 +21,6 @@
 
 public struct ValaLint.FormatMistake {
     public Check check;
-    public int line_index;
-    public int char_index;
+    public Vala.SourceLocation begin;
```

Comment:
Using 'begin' is possibly confusing as it is a Vala keyword - some expressions look a bit like an async function.  Grammatically, it should be `beginning` although `start` would be shorter.

Reply:
The name was copied from [Vala.SourceReference](https://github.com/GNOME/vala/blob/master/vala/valasourcereference.vala). I think we should stay as close as possible to Vala itself here.

Reply:
Oh, OK then.